gapipy/resources/base.py

```diff
@@ -55,6 +55,9 @@ def __repr__(self):
         else:
             return '<{}: {}>'.format(self.__class__.__name__, self.id)
 
+    def __hash__(self):
+        return hash(self.__repr__())
```

Comment:
Neat -- is it problematic, though, that an instance's `__repr__` changes whether or not you're holding a stub vs a fully hydrated instance?